src/components/EntityInfo/EntityMenu.vue

```diff
@@ -124,7 +127,10 @@
         <v-list dense>
           <v-list-item-group color="primary">
             <!-- View/Add Digital Credentials -->
-            <span v-if="isAllowed(AllowableActions.DIGITAL_CREDENTIALS)">
+            <span
+              v-if="isAllowed(AllowableActions.DIGITAL_CREDENTIALS) &&
+                IsAuthorized(AuthorizedActions.DIGITAL_CREDENTIALS)"
+            >
```

Comment:
This is good. I prefer keeping these checks separate instead of moving IsAuthorized into the isAllowed function.

FYI @bobdev-94

Reply:
Obsolete.